File: Lab5/C++/FigureLib/FigureLib/Circle.cpp
#define _USE_MATH_DEFINES
#include "Circle.h"
#include <cmath>

Circle::Circle(int radius) {
	this->radius = radius;
}

float Circle::CalcArea() {
	float area = M_PI * pow(radius, 2);
	return area;
}

float Circle::CalcPerimeter() {
	float perimeter = 2 * M_PI * radius;
	return perimeter;
}

File: Lab5/C#/FigureLib/FigureLib/Circle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FigureLib
{
    public class Circle : Figures
    {
        private float radius;

        public Circle(int radius)
        {
            this.radius = radius;
        }

        public override float calcArea()
        {
            float area = MathF.PI * MathF.Pow(radius, 2);
            return area;
        }

        public override float calcPerimeter()
        {
            float perimeter = MathF.PI * 2 * radius;
            return perimeter;
        }
    }
}
